File: NirvanaSMS.Api.Java/src/main/java/org/nirvanasms/api/core/entity/PhoneAndMessagePair.java
package org.nirvanasms.api.core.entity;

public class PhoneAndMessagePair {
	private PhoneNumber number;
    private String message;

    public PhoneAndMessagePair()
    {

    }

    public PhoneAndMessagePair(PhoneNumber number, String message)
    {
        this.number = number;
        this.message = message;
    }

    public PhoneAndMessagePair(String number, String message)
    {
        this.number = new PhoneNumber(number);
        this.message = message;
    }

	public PhoneNumber getNumber() {
		return number;
	}

	public void setNumber(PhoneNumber number) {
		this.number = number;
	}

	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}
}

File: NirvanaSMS.Api.CSharp/Core/Entity/PhoneAndMessagePair.cs
namespace NirvanaSMS.Api.Core.Entity
{
    public class PhoneAndMessagePair
    {
        public PhoneNumber Number { get; set; }
        public string Message { get; set; }

        public PhoneAndMessagePair()
        {

        }

        public PhoneAndMessagePair(PhoneNumber number, string message)
        {
            Number = number;
            Message = message;
        }

        public PhoneAndMessagePair(string number, string message)
        {
            Number = new PhoneNumber(number);
            Message = message;
        }
    }
}
